### ISSUE TITLE
The pipelines should have reporting mechanism through email

### ISSUE THREAD

**Issue by upadhyeammit** [transtats/transtats#279]
##### ISSUE TYPE

 - Feature Idea

##### OS / ENVIRONMENT / VERSION

No applicable 

##### SUMMARY

The pipelines should provide a way to send reports about the job run. Though users maybe concerned about failures its good to have selector for which kind of job report one wants to send. As like only failures, all or only successful.

The report should at least provide,
1. The failing job branch or resource name
2. Exception in failure
3. Language for which job failed

If possible also add links of,
1. The memsource of transifex resource and language so its easy to open that from email itself.

##### STEPS TO REPRODUCE

```

```

**Comment by sshedmake** [comment]
Yep, on reaching 100% Translation mark, an Email notification to required stakeholders i.e. Devs, QEs, Localization Program Managers, would be helpful
